tools/spectral/ipa/rulesets/IPA-124.yaml

```
@@ -0,0 +1,30 @@
+# IPA-124: Repeated Fields
+# http://go/ipa/124
+
+functions:
+  - IPA124ArrayMaxItems
+
+rules:
+  xgen-IPA-124-array-max-items:
+    description: |
+      Array fields must have a maxItems property defined to enforce an upper bound on the number of items (default: 100).
```

```suggestion
      Array fields must have a maxItems property defined to enforce an upper bound on the number of items (recommended max: 100). If repeated data has the chance of being too large, the API should use a sub-resource instead.
```
[Nit] Include guidance on option instead of very large array

---

I changed `repeated data` with `array field` to be consistent, and kept the rest. Let me know how it looks 👍